pkg/binfmt/Makefile

```diff
@@ -14,7 +15,8 @@ hash: $(DEPS)
 
 tag: hash
 	docker pull linuxkit/$(IMAGE):$(shell cat hash) || \
```

Comment:
this pull should have DCT too? Then we replace unsigned version with signed one. If its not signed its not real!